apps/datadogcharts/datadog_charts.star

```diff
@@ -0,0 +1,160 @@
+""")
+
+def main(config):
+    # Setup and validate config
+    DD_API_KEY = config.get("api_key") or DEFAULT_API_KEY
+    DD_APP_KEY = config.get("app_key") or DEFAULT_APP_KEY
+    DASHBOARD_ID = config.get("dashboard_id")
+
+    if DD_API_KEY == None or DD_APP_KEY == None:
+        child = render.Row(
+            cross_align = "center",
+            main_align = "center",
+            children = [
+                render.WrappedText(align = "center", content = "DataDog API Key or App Key not set."),
```

Comment:
Not critical right now, but later on it'd be great if the app could show a default graphic with mock data in this case. That's how we generate previews for apps, letting users know a bit about what to expect before they start digging up api keys and such.

Reply:
Happy to add it to the list for the second round of updates which I'm planning to work on next week.